### PR TITLE
Docs backport

### PR DESCRIPTION
**Pull request by montegoulding**
@asayd @livecodeali @seaniepie this patch is a back port of docs that should be just fixes. There's at least one patch to libURL included in this too but that's fairly benign (allowing no callback object specified on the auth callback) so should be ok

Anyway.. for those playing at home this patch was created by:

```
git log --pretty="%H" --cherry --reverse --diff-filter=M HEAD..develop -- *.lcdoc > ~/docspatches.txt
```
Then looking at the oneline version of the same log and removing all the hashes from `docspatches.txt` that at were new feature related. I also added some related patches that weren't picked up by git log.

Then:
```
cat ~/docspatches.txt | git cherry-pick -x -Xtheirs --stdin
```
As all these patches are already reviewed it's really just a matter of double-checking that there's nothing accidentally included that shouldn't be. If that's the case then we just need the hash and I can remove with an interactive rebase. A few eyes reviewing this would be good!

### CODE REVIEW

File: docs/dictionary/command/create.lcdoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ Synonyms: new
 
 Type: command
 
-Syntax: create [invisible] <objectType> [<objectName>] [in <group>]
+Syntax: create [invisible] <objectType> [<objectName>] [in {<group>|<card>}]
```
Comment (montegoulding) [inline docs/dictionary/command/create.lcdoc]: This one needs removal

File: docs/dictionary/command/difference.lcdoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
+Name: difference
+
```
Comment (montegoulding) [inline docs/dictionary/command/difference.lcdoc]: This one needs removal

File: docs/dictionary/command/intersect.lcdoc [hunk 3]
```diff
@@ -2,7 +2,7 @@ Name: intersect
 
 Type: command
 
-Syntax: intersect <targetArray> with <templateArray> [recursively]
+Syntax: intersect <targetArray> with <templateArray> [recursively] [into <destinationArray>]
```
Comment (montegoulding) [inline docs/dictionary/command/intersect.lcdoc]: This one needs removal

**Comment by montegoulding** [comment]
I have removed 4 errant commits but I think the rest are safe.

**Comment by montegoulding** [comment]
After merging this we could also back port the docs tests which will make it easier to recommend docs patches are made to `develop-X.X` rather than `develop`.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 61f26ab

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 61f26ab3ec679cd9e6e5f67bb192a9b287c8f59f

**Comment by livecode-vulcan** [comment]
😞 test failure 61f26ab3ec679cd9e6e5f67bb192a9b287c8f59f

* try-community-armv6-android-api8: [failure](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/629)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/645)
* try-community-js-emscripten-sdk1.35: [failure](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/619)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/562)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/565)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/623)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/643)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/628)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/645)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/627)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/638)
* try-community-x86-win32: [failure](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/660)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/624)

**Comment by montegoulding** [comment]
@livecode-vulcan retry 61f26ab

**Comment by livecode-vulcan** [comment]
😎 test success 61f26ab3ec679cd9e6e5f67bb192a9b287c8f59f

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/635)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/651)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/625)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/568)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/571)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/629)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/649)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/634)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/651)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/633)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/644)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/666)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/630)

**Pull request merged**